src/tools/attack/CMakeLists.txt

```diff
@@ -17,4 +17,7 @@ target_link_libraries(quicattack core platform)
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "MSVC")
     target_link_libraries(quicattack
         ws2_32 schannel ntdll bcrypt ncrypt crypt32 iphlpapi advapi32)
+    if(QUIC_UWP_BUILD)
+        target_link_libraries(quicattack OneCoreUAP)
```

Comment:
Just go ahead and add this to the list above unconditionally, assuming it works.